src/components/ToggleSwitch/TogleSwitch.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const ToggleLabel = styled.label`
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+  > input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+    &:before {
+      position: absolute;
+      content: '';
+      height: 26px;
+      width: 26px;
+      left: 4px;
+      bottom: 4px;
+      background-color: white;
+      transition: 0.4s;
+    }
+    &:checked + .slider {
+      background-color: #f85f73;
+    }
+    &:focus + .slider {
+      box-shadow: 0 0 1px #f85f73;
+    }
+    &:checked + .slider:before {
+      -webkit-transform: translateX(26px);
+      -ms-transform: translateX(26px);
+      transform: translateX(26px);
+    }
+  }
+  > .slider {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: #ccc;
+    transition: 0.4s;
+    &:before {
+      position: absolute;
+      content: '';
+      height: 26px;
+      width: 26px;
+      left: 4px;
+      bottom: 4px;
+      background-color: white;
+      -webkit-transition: 0.4s;
+      transition: 0.4s;
+    }
+    &.round {
+      border-radius: 34px;
+      &:before {
+        border-radius: 50%;
+      }
+    }
+  }
+`;
+
+const ToggleSwitch = () => (
+  <ToggleLabel htmlFor="switch-dark-mode" className="switch">
+    <input id="switch-dark-mode" type="checkbox" />
+    <span className="slider round" />
```

Comment:
This is good, however, since you are not re-using the classes on different elements (like in the Header component) it would make more sense to have these two elements as different styled components